src/create.cpp

```diff
@@ -815,6 +849,46 @@ namespace create {
       return 0;
     }
   }
+  
+  	  uint16_t Create::getCliffSignalLeft() const {
+	  if (data->isValidPacketID(ID_CLIFF_LEFT_SIGNAL)) {
```

Comment:
Thanks! I completely missed that. I assumed it was valid since it was defined on the headers. I'm going to try to add them to the protocols, but I won't have a Create 2 base to test them for at least a month. Maybe I should refrain from exposing the cliff signals until then.